internal/signaling/peer.go

```diff
@@ -108,6 +108,16 @@ func (p *Peer) HandlePacket(ctx context.Context, typ string, raw []byte) error {
 			return fmt.Errorf("unable to handle packet: %w", err)
 		}
 
+	case "leaveLobby":
```

Comment:
Is there no way we can just reuse "leave" or do we still think that will break some clients? Do we still receive the leave packet at all?

I feel it's a bit weird we suddenly have the _Lobby_ suffix for this packet type.

Reply:
Yeah we should be ably to use leave by now, it has been 2 years. Also worst case a client will leave the lobby instead of completely shutting down the network. I haven't seen any games use proper shutdown yet anyways as they all just wait for the tab to be closed.